Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -44,6 +43,23 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
+		public static readonly BindableProperty RemainingItemsThresholdReachedCommandProperty = BindableProperty.Create(nameof(RemainingItemsThresholdReachedCommand), typeof(ICommand), typeof(ItemsView), null);
+
+		public ICommand RemainingItemsThresholdReachedCommand
+		{
+			get { return (ICommand)GetValue(RemainingItemsThresholdReachedCommandProperty); }
+			set { SetValue(RemainingItemsThresholdReachedCommandProperty, value); }
+		}
+
+		public static readonly BindableProperty RemainingItemsThresholdProperty =
+			BindableProperty.Create(nameof(RemainingItemsThreshold), typeof(int), typeof(ItemsView), -1, validateValue: (bindable, value) => (int)value >= -1);
```

Comment:
setting the value to -1 shouldn't be allowed. `ClearValue` is there for that

Reply:
There are many examples in the code where setting the value to default is allowed. Do you still want me to change this?